assets/displayStats.js

```diff
@@ -0,0 +1,14 @@
+export function displayStats(userMistakesCount, userKeyTypeCount, seconds) {
+  if (userMistakesCount == 0 && userKeyTypeCount == 0) {
+    return 'CPM: ' + '0 ' + 'Wrong Chars: ' + '0%';
+  }
+
+  seconds = seconds / 60;
```

Comment:
Uhm what?

Reply:
I compute the minutes out of the seconds in order to determine the CPM. Thus the seconds are always handed over by parameter, the second count is a number without the division through 60.

Reply:
```suggestion
  let minutes = seconds / 60;
```